markdown-links-verifier-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "excludeStartingWith": [
+    "xref:",
+    "~/",
+    "/",
+  ]
```

Comment:
I had to use `~/` for `_csharplang` links instead of `../../<etc>/_csharplang` because there is no "excludeContains" option. Let me know if you don't prefer to use `~/_csharplang` and I can introduce a new option in the GH Action.

Note: One of the benefits of using `~/csharplang` is it's not affected if the file is moved between different directories. Another small benefit is being consistent since there are [lots of links](https://grep.app/search?q=~/_csharplang&filter[repo][0]=dotnet/docs) already use `~/csharplang`.

Reply:
The downside is that `~` links aren't verified by build. I'd prefer to leave them as file-relative links.

Reply:
@gewarren The GH Action won't be able to validate cross-repo links anyway. If I converted them to file-relative links, they will be flagged as invalid. I'll try working on cross-repo validation if this is something you want.